README.md

```diff
@@ -1,30 +1,7 @@
-Kafka Scala API Tester - Advanced Nagios Plugin / CLI Tool with Kerberos support
+Kafka Scala API Tester - Advanced Nagios Plugin / CLI Tool with Kerberos support (CMM customized)
```

Comment:
if we want to get the change merged in I would back this change out

Reply:
are all of the changes to readme.md still valid?

Reply:
I'm not considering merging our fork back to upstream. The changs here are all serving cmm purpose.

Reply:
well, my use of kafka has kerberos as a requirement so ;)